regexes/device/mobiles.yml

```diff
@@ -39485,16 +39509,16 @@ Venstar:
   device: 'tablet'
   model: '$1'
 
-# Nothing Phone
+# Nothing Phone or CMF Phone (https://cmf.tech/)
```

Comment:
```suggestion
# Nothing Phone (https://nothing.tech/)
```

I think it should be created a new brand, `CMF`.

_CMF, a sub-brand of Nothing_ - https://cmf.tech/pages/about-us